src/_pytest/_code/__init__.py

```diff
@@ -17,6 +18,7 @@
     "getfslineno",
     "getrawcode",
     "Traceback",
+    "TracebackEntry",
```

Comment:
`TracebackEntry` is exposed through `Traceback` so it should be visible too (though this is an internal package anyway).